IdentitySdkFacebook/Podfile

```diff
@@ -4,6 +4,6 @@ platform :ios, '12.2'
 
 target 'IdentitySdkFacebook' do
   pod 'IdentitySdkCore', :path => '../IdentitySdkCore.podspec'
-  pod 'FBSDKCoreKit'
-  pod 'FBSDKLoginKit'
+  pod 'FBSDKCoreKit', '~> 9.0.0'
```

Comment:
Commit 6ba18f7cd25edb5d275351c8bfff8bb10080cf59 message says you downgraded this (from 13.x), what prompted this ?

Reply:
I don't want to upgrade this lib in this commit.
The Podfile used to declare the dependency like this : `pod 'FBSDKLoginKit'`, without version number.
It instructs cocoa pods to take the latest version available.
It means that when I launched `pod update` the dependency moved automatically from 9.0.0 to 13.2.
So I moved it back to 9 because there are some breaking changes.
I'll address them in the dedicated PR for upgrading dependencies.

PS: I should have just declared the dependency like this `pod 'FBSDKCoreKit', '9.0.0'` to avoid the patch number bump.